extension/models/catalog/rootCategories/rootCategories.js

```diff
@@ -0,0 +1,28 @@
+const Category = require('../category/category')
+
+/**
+ * RootCategories Model
+ */
+class RootCategories extends Category{
+  constructor() {
+    super()
+  }
+
+  /**
+   * @param {Array} categories
```

Comment:
At this point it's clear what kind of array is returned, and it should be stated as such.

Reply:
Done